Source/WebKit/NetworkProcess/webtransport/NetworkTransportStream.h

```diff
@@ -53,8 +55,10 @@ class NetworkTransportStream : public RefCounted<NetworkTransportStream>, public
 
     WebCore::WebTransportStreamIdentifier identifier() const { return m_identifier; }
 
-    void sendBytes(std::span<const uint8_t>, bool withFin);
-
+    void sendBytes(std::span<const uint8_t>, bool, CompletionHandler<void(std::optional<WebCore::Exception>&&)>&&);
+    void cancelReceive(std::optional<unsigned>);
```

Comment:
Are error codes 32 bits?  If so, I think we should do something like this:
using NetworkTransportErrorCode = uint32_t
Otherwise it's a little unclear what the unsigned is.